tests/system/providers/microsoft/azure/example_synapse_run_pipeline.py

```diff
@@ -36,15 +36,15 @@
     catchup=False,
     tags=["synapse", "example"],
 ) as dag:
-    begin = EmptyOperator(task_id="begin")
-
+   begin = EmptyOperator(task_id="begin")
```

Comment:
```suggestion
    begin = EmptyOperator(task_id="begin")
```

You changed the indent of this line thus it causes tests to fail.
This will fix the problem